src/styles/blocks/page.css

```
@@ -0,0 +1,14 @@
+.page {
```

it's better to set for the whole `html`
```suggestion
html {
```